Week6/app/src/main/java/com/oliverperez/blockbusterchallenge/model/MovieRepository.kt

```diff
@@ -1,10 +1,8 @@
 package com.oliverperez.blockbusterchallenge.model
 
 import androidx.lifecycle.LiveData
-import com.oliverperez.blockbusterchallenge.model.Movie
 
 interface MovieRepository {
-    fun addMovie(movie: Movie)
     fun getAllMovies(): LiveData<List<Movie>>
     fun clearAllMovies()
```

Comment:
When working with coroutines you want to make these `suspend` functions. That way, when you call them from the ViewModel class they already provide suspend behavior and you don't need to worry about it.